[2025-02-08T19:00:11.933+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:00:11.944+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2025-02-08T19:00:11.951+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2025-02-08T19:00:11.969+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:00:11.982+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-10-09 00:00:00+00:00
[2025-02-08T19:00:11.994+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-10-09T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpr4rr5wwd']
[2025-02-08T19:00:11.997+0000] {standard_task_runner.py:105} INFO - Job 33: Subtask generate_account_dim_data
[2025-02-08T19:00:11.997+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=579) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:00:12.001+0000] {standard_task_runner.py:72} INFO - Started process 592 to run task
[2025-02-08T19:00:12.075+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-10-09T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:00:12.194+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-09T00:00:00+00:00'
[2025-02-08T19:00:12.195+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:00:12.214+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T19:00:12.214+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:00:12.222+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:00:12.223+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-10-09T00:00:00+00:00, execution_date=20241009T000000, start_date=20250208T190011, end_date=20250208T190012
[2025-02-08T19:00:12.256+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:00:12.270+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:00:12.272+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:39:59.136+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:39:59.156+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2025-02-08T20:39:59.170+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-10-09T00:00:00+00:00 [queued]>
[2025-02-08T20:39:59.171+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:39:59.195+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-10-09 00:00:00+00:00
[2025-02-08T20:39:59.211+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=894) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:39:59.212+0000] {standard_task_runner.py:72} INFO - Started process 911 to run task
[2025-02-08T20:39:59.213+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-10-09T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpo11nxp6j']
[2025-02-08T20:39:59.216+0000] {standard_task_runner.py:105} INFO - Job 78: Subtask generate_account_dim_data
[2025-02-08T20:39:59.291+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-10-09T00:00:00+00:00 [running]> on host 7e959c1a2a8b
[2025-02-08T20:39:59.436+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-09T00:00:00+00:00'
[2025-02-08T20:39:59.442+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:39:59.472+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T20:39:59.473+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:39:59.482+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:39:59.483+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-10-09T00:00:00+00:00, execution_date=20241009T000000, start_date=20250208T203959, end_date=20250208T203959
[2025-02-08T20:39:59.593+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:39:59.620+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:39:59.625+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
